docs/src/main/asciidoc/logging.adoc

```diff
@@ -526,7 +523,7 @@ quarkus.log.console.filter=my-filter
 
 == Logging configurations examples
 
-This chapter provides examples of frequently used logging configurations.
+Frequently used logging configurations examples.
```

Comment:
Not a sentence at all?

Reply:
What about:
Obtain the proper logger settings from the frequently used logging configurations examples.

Reply:
Frankly, there's no need for an abstract here. The heading says it perfectly. But if we need something, how about this?

> Following are examples of frequently used logging configuration.

I know we don't like "follow" or "in this chapter" or so, but how exactly can we describe a collection of examples? :-)

Reply:
Thats another bad influence of playing Diablo 4 with all these typos and misconceptions :)

Reply:
I used:

```
Obtain the proper logger settings from the frequently used logging configurations examples.
```
Can imagine this line being used as a section description TBH.
Close the thread if OK.

@michelle-purcell , or do you have any better idea how to solve this? :)

Reply:
I'd say we can obtain inspiration from examples, but often not "proper settings" :-) Would be good to have something better, for sure.

Reply:
```
Get inspiration for real-world logger settings from frequently used logging configuration examples.
```
:)